ingestion/functions/parsing/india/india.py

```diff
@@ -78,63 +93,47 @@ def convert_demographics(row):
 
 
 def convert_sources(row):
+    # Sources must be unique, per our case schema.
+    included = set()
     additionalSources = [{"sourceUrl": row[col]}
                          for col in ["Source_1", "Source_2", "Source_3"]
-                         if row[col]]
+                         if row[col] and
+                         row[col] not in included and not included.add(
+                             row[col])]
     return additionalSources or None
 
 
 def update_for_outcome(row, case):
-    if row["Current Status"] == "Recovered":
+    if row["Current Status"] == "Hospitalized":
         case["events"].append({
-            "name": "outcome",
+            "name": "hospitalAdmission",
             "dateRange":
             {
                 "start": convert_date(row["Date Announced"]),
                 "end": convert_date(row["Date Announced"])
             },
-            "value": "Recovered"
+            "value": "Yes"
         })
-    elif row["Current Status"] == "Deceased":
```

Comment:
Super unimportant, was the change from 'elif' to 'if' important? I can't see that it makes a difference, and certainly I'm not suggesting it needs changing back, just interested if this was a motivated or autopilot change. You still have elif down in l127.

Reply:
Just an autopilot mishap, apologies. These are mutually exclusive conditions, so despite the functional lack of difference, my intention was to use `elif`.